packages/get-platform/src/getPlatform.ts

```diff
@@ -282,7 +287,7 @@ export async function getPlatform(): Promise<Platform> {
 
   // use the debian build with OpenSSL 1.1 as a last resort
   // TODO: perhaps we should default to 'debian-openssl-3.0.x'
-  return 'debian-openssl-1.1.x'
+  return `debian-openssl-${defaultLibssl}`
```

Comment:
Maybe this makes the change a bit clearer for the untouched code?
```diff
  // if just OpenSSL is known, fallback to debian with a specific libssl version
  if (osResult.libssl) {
-    return ('debian-openssl-' + osResult.libssl) as Platform
+    return ('debian-openssl-' + osResult.libssl) as Platform
  }
```

Reply:
I've followed a similar advice to make the default more explicit, see https://github.com/prisma/prisma/pull/17151/commits/6b9dfb64b7bc632a174ab2b0ada71f70c2132b1f